internal/attestation/renderer/chainloop/v02.go

```diff
@@ -165,9 +169,43 @@ func (r *RendererV02) predicate() (*structpb.Struct, error) {
 		return nil, fmt.Errorf("error unmarshaling predicate: %w", err)
 	}
 
+	// Policies have oneofs, which are not compatible with the regular json.Marshal. We need to do it separately
+	policyValues, err := policiesToValues(r.att.Policies)
+	if err != nil {
+		return nil, fmt.Errorf("error converting policies to values: %w", err)
+	}
+	// Store it in a ListValue struct.
+	predicate.Fields["policies"] = &structpb.Value{Kind: &structpb.Value_ListValue{ListValue: &structpb.ListValue{Values: policyValues}}}
```

Comment:
do you need to store the result as structpb? I thought we had a custom predicate that we were mapping ourselves?

Reply:
Just because we have some `oneof`s in the policies, which is incompatible with structpb conversion.